surveys/2021-trademark-survey/questions.md

```diff
@@ -50,9 +51,9 @@ are used for. Trademark law also covers confusion about whether the trademark
 owner is endorsing or sponsoring the user. For example, when websites include
 their customers’ logos on their websites, we understand that the customer’s
 logo is being used to signal that the owner of the logo approves of or endorses
-the product, and we take that information into account when considering the
-product. A suggestion of endorsement that is inaccurate can also be a trademark
-infringement.
+the product and possibly the content of the website, and we take that
+information into account when considering the product and website. A suggestion
```

Comment:
```suggestion
information into account when considering the owner of the website. A suggestion
```

Reply:
These two sentences read odd to me and this is my attempt to maybe make it read less odd.

Reply:
Seems reasonable to me, in the same spirit; any of those endorsements could be possible.